crates/typst-library/src/model/heading.rs

```diff
@@ -179,6 +179,16 @@ pub struct HeadingElem {
     #[default(Smart::Auto)]
     pub hanging_indent: Smart<Length>,
 
+    /// The gap between the numbering and the start of the heading.
+    /// Default value is `{0.3em}`.
```

Comment:
I think default values are displayed in the documentation. So this is redundant.